runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsConfig.java

```diff
@@ -54,14 +56,25 @@ default boolean enableSkillIcon()
 
 	@ConfigItem(
 		keyName = "enableRestorationBars",
-		name = "Show restores",
-		description = "Visually shows how much will be restored to your status bar."
+		name = "Show amount of hitpoints and prayer restored",
```

Comment:
Usually we prefer short names since there is not much room for it in the side panel. When you hover over it it will show the description anyway. This would probably just show `Show amount...`

Reply:
Agreed! Change made.